src/Symfony/Component/Notifier/Bridge/LightSms/LightSmsTransport.php

```diff
@@ -0,0 +1,172 @@
+        '000' => 'Service unavailable',
+        '1' => 'Missing Signature',
+        '2' => 'Login not specified',
+        '3' => 'Text not specified',
+        '4' => 'Phone number not specified',
+        '5' => 'Sender not specified',
+        '6' => 'Invalid signature',
+        '7' => 'Invalid login',
+        '8' => 'Invalid sender name',
+        '9' => 'Sender name not registered',
+        '10' => 'Sender name not approved',
+        '11' => 'There are forbidden words in the text',
+        '12' => 'Error in SMS sending',
+        '13' => 'Phone number is in the blackist. SMS sending to this number is forbidden.',
+        '14' => 'There are more than 50 numbers in the request',
+        '15' => 'List not specified',
+        '16' => 'Invalid phone number',
+        '17' => 'SMS ID not specified',
+        '18' => 'Status not obtained',
+        '19' => 'Empty response',
+        '20' => 'The number already exists',
+        '21' => 'No name',
+        '22' => 'Template already exists',
+        '23' => 'Missing Month (Format: YYYY-MM)',
+        '24' => 'Timestamp not specified',
+        '25' => 'Error in access to the list',
+        '26' => 'There are no numbers in the list',
+        '27' => 'No valid numbers',
+        '28' => 'Missing start date (Format: YYYY-MM-DD)',
+        '29' => 'Missing end date (Format: YYYY-MM-DD)',
+        '30' => 'No date (format: YYYY-MM-DD)',
+        '31' => 'Closing direction to the user',
+        '32' => 'Not enough money',
+        '33' => 'Missing phone number',
+        '34' => 'Phone is in stop list',
+        '35' => 'Not enough money',
+        '36' => 'Can not obtain information about phone',
+        '37' => 'Base Id is not set',
+        '38' => 'Phone number already exists in this database',
+        '39' => 'Phone number does not exist in this database',
+    ];
+
+    public function __construct(string $login, string $password, string $from, HttpClientInterface $client = null, EventDispatcherInterface $dispatcher = null)
+    {
+        $this->login = $login;
+        $this->password = $password;
+        $this->from = $from;
+
+        parent::__construct($client, $dispatcher);
+    }
+
+    public function __toString(): string
+    {
+        return sprintf('lightsms://%s/external/get/send.php?from=%s', $this->getEndpoint(), $this->from);
+    }
+
+    public function supports(MessageInterface $message): bool
+    {
+        return $message instanceof SmsMessage;
+    }
+
+    protected function doSend(MessageInterface $message): SentMessage
+    {
+        if (!$message instanceof SmsMessage) {
+            throw new UnsupportedMessageTypeException(__CLASS__, SmsMessage::class, $message);
+        }
+
+        $timestamp = time();
+
+        $signature = $this->generateSignature($message, $timestamp);
+
+        $endpoint = sprintf(
+            'https://www.%s/external/get/send.php?login=%s&signature=%s&phone=%s&text=%s&sender=%s&timestamp=%s',
```

Comment:
```suggestion
            'https://%s/external/get/send.php?login=%s&signature=%s&phone=%s&text=%s&sender=%s&timestamp=%s',
```
because HOST constant already includes www?

Reply:
Nice catch

Reply:
Yes. We cannot remove it because core tests fail after this change

Reply:
Yes, but in this case it can be removed where we build `$endpoint`

Reply:
I see. My bug. Thanks @norkunas!